### PR TITLE
Issue 197. Fix handling of optional package fields

### PR DESCRIPTION
**Pull request by AlexanderYukhanov**

### CODE REVIEW

**Comment by goneall** [comment]
It looks like your editor reformatted much of the changed files.

Could you make the changes without the changes to the white space?

FYI - We're using the default Eclipse 2.1 formatter which is set to tabs only, tab size of 4.

**Comment by AlexanderYukhanov** [comment]
updating

**Comment by AlexanderYukhanov** [comment]
Restored the surrounding formatting, please take another look

File: src/main/java/org/spdx/library/model/SpdxPackage.java [hunk 1]
```diff
@@ -606,19 +603,13 @@ protected List<String> _verify(Set<String> verifiedIds, String specVersion) {
 		
 		// files depends on if the filesAnalyzed flag
 		try {
-			if (getFiles().size() == 0) {
-				if (filesAnalyzed) {
```
Comment (goneall) [inline src/main/java/org/spdx/library/model/SpdxPackage.java:610]: I noticed we removed the check for files being present when files analyzed is true.

I went back and read the spec, and indeed there is no stated requirement that files be present - so this change is correct.

**Review by goneall — APPROVED** [review]
Thanks @AlexanderYukhanov 

I'll merge this in for the next release.

**Pull request merged**